srcpkgs/atf-rk3399-bl31/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'atf-rk3399-bl31'
+pkgname=atf-rk3399-bl31
```

Comment:
I think we should call this package arm-trusted-firmware, so we can extend it to other platforms later on.

Reply:
I would rather not have updates to one hardware platform affect the others. And each build is independent anyway.

Reply:
Can we merge into single package and use sub-packages?

Reply:
We could but it would still mean tying versions of one platform to the other platforms. It is a question of whether the distribution wants to update all of the TF-A binaries at the same time (and presumably all of the u-boot binaries as well, otherwise why bother?), or whether the builds should be kept separate and updated independently. I lean toward the latter because I do not have access to all of the hardware platforms that might be affected by a new release of TF-A. I would feel uncomfort

Reply:
Can I consider this thread resolved?